http/client_collection.go

```diff
@@ -125,6 +130,13 @@ func (c *Collection) CreateMany(
 	return nil
 }
 
+func setDocEncryptionHeaderIfNeeded(ctx context.Context, req *http.Request) {
```

Comment:
todo: This should be merged into `setDefaultHeaders` func in `http_client.go`

Reply:
all right